doc/en/user/source/extensions/vectortiles/options.rst

```diff
@@ -0,0 +1,108 @@
+.. _vectortiles.options:
+
+Vector Tiles Generation Options
+-------------------------------
+
+The content of the vector tiles generated by this output format is driven by the
+style used in the GetMap/GetTile request.
+
+The style can be pretty basic, the vector tile output format considers only the following:
+
+* Rules scale denominators, and the rule filters, control which features are included in the output
+  (e.g., in a road layer, one can include only highways at low zoom levels)
+* Feature type style and rule vendor options can provide more fine grained control over the output. 
+  Inclusion of options in the rule is recommended only when trying to apply the options in a  scale dependent way.
+
+In particular, the following vendor options can be included:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Option
+     - Description
+   * - vt-attributes
+     - A comma separated list of attributes to be included in the output. Reducing the attributes helps control the tile size.
+   * - vt-coalesce
+     - ``true``/``false``, enable feature geometry merging, when attribute values are equal. This can greatly reduce the tile size,
+       especially when coupled when a small attribute list, which helps finding more attribute duplicates.
+       To perform duplicate detection the code will run a "sort" on the attributes, if the source is a database, best to keep the
+       indexed ones first in the list.
```

Comment:
- ``true``/``false``, enables feature geometry merging when attribute values are equal. This can greatly reduce the tile size, especially when coupled with a small attribute list, which helps in finding more attribute duplicates.
       To perform duplicate detection, the code will run a ‘sort’ on the attributes. If the source is a database, it is best to keep the indexed ones first in the list.